app/Http/Controllers/ProjectController.php

```diff
@@ -16,31 +16,31 @@ class ProjectController extends Controller
      */
     public function index(Request $request)
     {
-        if ($request->get('pid')) {
-            $projects = Project::find(16);
-            $projects->members->pluck('id')->toArray();
+      if ($request->get('pid')) {
+        $projects = Project::find(16);
+        $projects->members->pluck('id')->toArray();
```

Comment:
You are not assigning this to a variable so you can remove it or assign it to a variable and use it.